codex_of_the_damned/templates/archetypes/bibliodeque/prov_nav.py

```diff
@@ -0,0 +1,15 @@
+Nav(
+    lazy_gettext("Bibliodèque"),
+    children=[
+        Nav(lazy_gettext("Lorrie Superstar")),
+        Nav(lazy_gettext("Howler and Friends")),
+        Nav(lazy_gettext("Muricia and Friends")),
+        Nav(lazy_gettext("Akunanse Classic")),
+        Nav(lazy_gettext("Tunnel Runners")),
+        Nav(lazy_gettext("Army of Allah")),
+        Nav(lazy_gettext("Tariq Eats the World")),
+        Nav(lazy_gettext("Cybele Maleficia")),
+        Nav(lazy_gettext("He-Who-Shall-Not-Be-Named")),
```

Comment:
The unnamed archetype is already in the archetypes list - are we ok with duplicates ?